context/app/static/js/components/entity-search/Search/utils.js

```diff
@@ -15,4 +15,12 @@ function buildSortPairs(fields) {
     .flat();
 }
 
-export { buildSortPairs };
+function getNumericFacetProps(fieldName, numericFacetsProps) {
+  if (!numericFacetsProps?.[fieldName]) {
+    return {};
+  }
+
+  return { range: numericFacetsProps[fieldName] };
```

Comment:
Maybe a naming tweak for consistency? Either to the function name, or to this parameter: Right now `numericFacetsProps` might be either the _entire_ returned object, or just the value of `range`.

Reply:
Renamed the function, thanks!